src/PhpSpec/Console/ResultConverter.php

```diff
@@ -21,6 +21,23 @@
 class ResultConverter
 {
     /**
+     * @var IO
+     */
+    private $io;
+
+
```

Comment:
Do you mean there is an extra one?  I'm guessing 1 line between variables & method definition then?